src/jit/flowgraph.cpp

```diff
@@ -4261,33 +4261,6 @@ void        Compiler::fgFindJumpTargets(const BYTE * codeAddr,
         pSm->Start(this);
     }
 
-#ifdef FEATURE_LEGACYNETCF
```

Comment:
Are you sure we should delete this? We still build with this enabled (see the root CMakeLists.txt)